app/views/users/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<section class="splash">
+  <h1>EveryKwacha💸</h1>
+  <%= render partial: "devise/shared/notification"%>
+  <div class="regBtns">
+    <%= link_to "LOG IN", new_user_session_path, data: {turbo: false}, class: "buttons" %>
+    <%= link_to "SIGN UP", new_user_registration_path, data: {turbo: false}, class: "buttons" %>
+  </div>
+</section>
```

Comment:
- [ ] I can't test the application due to the routes that do not exist in the application, please make sure that all the routes for the login and register are working properly in order to test the application.

![image](https://user-images.githubusercontent.com/55635977/157733962-097ac80f-5179-4a28-886e-fbe19e4d8025.png)

![image](https://user-images.githubusercontent.com/55635977/157734095-543536c1-4dcf-4e60-b219-fb1d600de9c9.png)